src/reference/asciidoc/smb.adoc

```diff
@@ -171,6 +186,138 @@ public class SmbJavaApplication {
 ----
 ====
 
+[[smb-streaming-inbound]]
+=== SMB Streaming Inbound Channel Adapter
+
+This adapter produces message with payloads of type `InputStream`, letting files be fetched without writing to the local file system.
+Since the session remains open, the consuming application is responsible for closing the session when the file has been consumed.
+The session is provided in the `closeableResource` header (`IntegrationMessageHeaderAccessor.CLOSEABLE_RESOURCE`).
+Standard framework components, such as the `FileSplitter` and `StreamTransformer`, automatically close the session.
+See <<./file.adoc#file-splitter,File Splitter>> and <<./transformer.adoc#stream-transformer,Stream Transformer>> for more information about these components.
+The following example shows how to configure an `inbound-streaming-channel-adapter`:
+
+====
+[source, xml]
+----
+<int-smb:inbound-streaming-channel-adapter id="smbInbound"
```

Comment:
But this one is also missed in the XSD...

As I said: we can add an XML support for them or just omit it and go ahead with only Java & Annotations or DSL configuration approach.
But doc anyway has to reflect the current reality to be merged.

Reply:
Yes of course, as I don't have experience with XML configuration in general as I always use Java & Annotations or DSL configuration approach, I will omit those XML configuration examples.